tools/ms2snoop/MS2snoop.xml

```diff
@@ -48,9 +48,15 @@
     --ionization '$sirius.ionization'
     --fragment_match_delta '$sirius.fragment_match_delta'
     --fragment_match_delta_unit '$sirius.fragment_match_delta_unit'
-    $advenced.pdf
+    #if $advenced.pdf:
+        --pdf_path './out.zip'
+    #end if
     $advenced.debug
     $advenced.verbose
+#if $advenced.pdf:
+&& cat ./out.zip > $pdf_output
```

Comment:
pdf_output needs to be single quoted for security reasons: https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/integration_checklist.html?highlight=quotes#creating-the-tool-wrapper-xml-file

Reply:
Yes, you're right, thanks!